### ISSUE TITLE
Cannot set metadata

### ISSUE THREAD

**Issue by marufbd** [kakone/GoogleCast#16]
Setting metadata throws exception:
```
var mediaInfo = new MediaInformation
                        {
                            ContentId = link,
                            ContentType = mimeType,
                            StreamType = StreamType.Buffered,
                            Metadata = new GenericMediaMetadata
                            {
                                Title = "Sample",
                                Subtitle = "subtitle",
                                Images = new[]
                                {
                                    new Image{Url = thumbUrl}
                                }
                            }
                        };
```

**Comment by kakone** [comment]
Fixed in [version 1.6.5](https://www.nuget.org/packages/GoogleCast/1.6.5).

**Comment by marufbd** [comment]
Nice work! Thanks.
